src/hexo/themes/sonarwhal/layout/partials/metas.hbs

```diff
@@ -1,3 +1,6 @@
+<meta property="og:image" content="{{setDefault page.image '/images/sonar-logo.svg'}}" />
```

Comment:
According to [this](https://indieweb.org/The-Open-Graph-protocol#Does_not_support_SVG_images), SVG is not supported.

Reply:
I was thinking about using the apple-touch-icon